spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala

```diff
@@ -165,7 +168,14 @@ class CheckpointedDrmSpark[K: ClassTag](
       else if (classOf[Writable].isAssignableFrom(ktag.runtimeClass)) (x: K) => x.asInstanceOf[Writable]
       else throw new IllegalArgumentException("Do not know how to convert class tag %s to Writable.".format(ktag))
 
-    rdd.saveAsSequenceFile(path)
```

Comment:
Ok, need to rebase against your branch. 

If I can get this to build against the current master/0.11.0-SNAPSHOT I'd be inclined to push it so someone speak up if there is an objection. Once again, it will disable the shell build, for now.